packages/venia-ui/lib/components/CartPage/cartPage.css

```diff
@@ -0,0 +1,76 @@
+.root {
+    padding: 1.5rem;
+}
+
+.body {
+    display: grid;
+    gap: 2rem;
+    grid-template-columns: 66% auto;
+    /* The summary grid item spans the entire right column. */
+    grid-template-areas:
+        'items summary'
+        'adjustments summary'
+        'recents summary';
+}
+
+.heading_container {
+    display: flex;
+    align-items: baseline;
+    justify-content: space-between;
+
+    margin-bottom: 1rem;
+}
+
+.heading {
+    line-height: 1.25em;
+}
+
+.items_container {
+    grid-area: items;
+}
+
+.price_adjustments_container {
+    grid-area: adjustments;
+}
+
+.recently_viewed_container {
+    grid-area: recents;
+}
+
+.sign_in {
+    font-weight: bold;
+    text-decoration: underline;
```

Comment:
This text looks a little larger than in the mocks, at least in relation to its counterpart in the header, "Cart".

Reply:
That's actually a good callout - I don't know how to open the mock in XD to get the correct size. I'll dig deeper.

Reply:
Should be able to find a link somewhere in the epic or linked issues. But @tjwiebell is also having trouble finding pixel/size direction from XD so not sure how to help from there.